reference/tools/cmake/cmake_layout.rst

```diff
@@ -87,18 +90,3 @@ The ``CMakePresets.json`` file generated at the :ref:`CMakeToolchain<conan-cmake
 generator, will also take this ``tools.cmake.cmake_layout:build_folder_vars`` config into account to generate different
 names for the presets, being very handy to install N configurations and building our project for any of them by
 selecting the chosen preset.
-
-
-.. note::
```

Comment:
This removal is not related with this PR, but something that was outdated: https://github.com/conan-io/docs/pull/2944/files